README.md

```diff
@@ -28,23 +28,69 @@ See the official [React Native website](https://facebook.github.io/react-native/
 
 *Note*: Development on React Native Windows itself currently requires Visual Studio 2017. It is not supported with VS Code, but we will gladly accept pull requests to enable a great developer experience in those environments.
 
 ## Getting Started
 
-- Follow the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) to install React Native and its dependencies.
-- Change your working directory to the project directory you specified in the `react-native init` command from the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) (e.g., `cd AwesomeProject`).
-- Install [rnpm-plugin-windows](https://www.npmjs.com/package/rnpm-plugin-windows) from NPM (i.e., `npm install --save-dev rnpm-plugin-windows` or `yarn add rnpm-plugin-windows --dev` if you have Yarn installed).
-- Initialize Windows project with `react-native windows` command in the project directory.
-- *Note*: You will need to install the [requirements](#system-requirements) before you can build and run your application.
+### Installing React Native and its dependencies
+This is a targeted summary of setup React Native setup for specifically React Native Windows. See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for full details.
+
+#### Dependencies: Node, Python, JDK
+Install the dependencies [specified by React Native](http://facebook.github.io/react-native/docs/getting-started.html#node-python2-jdk). Specifically, make sure a recent version of Node, Python2, and a recent version of the Java SE Development Kit (JDK) are installed.
+
+[Chocolatey](https://chocolatey.org/) is the React Native recommended installation method. On an elevated Command Prompt, run:
+```
+choco install -y nodejs.install python2 jdk8
+```
+
+#### React Native CLI
+Next, we need the React Native command line interface. Install using NPM:
+```
+npm install -g react-native-cli
+```
+
+#### Initialize your project
+Next, [generate a React Native project](http://facebook.github.io/react-native/docs/getting-started.html#creating-a-new-application). In the directory you would like your React Native Windows project directory, run:
+```
+react-native init <project name>
+```
+Navigate into this newly created directory:
+```
+cd <project name>
+```
+
+### Installing React Native Windows
+#### React Native Windows CLI
+Now we need to install the React Native Windows command line interface ([rnpm-plugin-windows](https://www.npmjs.com/package/rnpm-plugin-windows)).
+
+If you are using NPM, run
+```
+npm install --save-dev rnpm-plugin-windows
+```
+
+If you are using Yarn, run
+```
+yarn add rnpm-plugin-windows --dev
+```
+#### Initialize your project
+Initialize your React Native Windows project in the project directy by running:
+```
+react-native windows
+```
+
+*Note*:  You will need to install the [requirements](#system-requirements) before you can build and run your application.
+
```

Comment:
*Pedantic*: We should be consistent in the markdown files if there is a trailing newline between sections.